synth.py

```diff
@@ -0,0 +1,29 @@
+import synthtool as s
```

Comment:
I'm a little bit hesitant to include the tool with (a) no license, (b) no documentation, (c) no ideas how to use it.

Reply:
license header is required and should have been added. As far as instructions I am going to hold off since we are working on moving where this lives and I will be going across our repositories to add details once that lands.